quickstarts/post-configure-example/README.md

```diff
@@ -0,0 +1,200 @@
+----
+This empty file is needed until the following jira is not fixed:
+https://issues.redhat.com/browse/RHPAM-3665
+
+postconfigure.sh:
+----
+echo "******  RUNNING ADDITIONAL CONFIGURATIONS WITH JBOSS-CLI - ADDING EXTRA ELYTRON USERS TO KIE FS REALM **********"
+echo "trying to execute /opt/eap/bin/jboss-cli.sh --file=/opt/eap/extensions/add-users.cli"
+/opt/eap/bin/jboss-cli.sh --file=/opt/eap/extensions/add-users.cli
+echo "END - users added"
+
+Events:  <none>
+```
+
+
+With the config-map created, you can now proceed with the container configuration.
+In this example there is only one KIE Server running. The steps will be described by using the `operator` and 
+`application templates`.
+
+
+### Application Templates method:
+
+When using the `application templates` or an already running RHPAM instance, it turns to be a little easier to 
+configure than using `operators`.
+
+With the config map created, mount it as a `volume` on the desired Deployment Config:
+
+```bash
+$ oc set volumes dc/myapp-kieserver --add  \
+    --configmap-name=postconfigure \
+    --mount-path=/opt/eap/extensions \
+    --default-mode=0555
+```
+
+Note, if the target *dc* is other than `myapp-kieserver` remember to update it to fit your needs.
+This change will make the running KIE Server be redeployed, after it starts again you should see a message similar to:
+
+```bash
+******  RUNNING ADDITIONAL CONFIGURATIONS WITH JBOSS-CLI - ADDING EXTRA ELYTRON USERS TO KIE FS REALM **********
+trying to execute /opt/eap/bin/jboss-cli.sh --file=/opt/eap/extensions/add-users.cli
+...
+some jboss logging output
+...
+The batch executed successfully
+15:33:07,651 INFO  [org.jboss.as] (MSC service thread 1-2) WFLYSRV0050: JBoss EAP 7.4.1.GA (WildFly Core 15.0.4.Final-redhat-00001) stopped in 40ms
+END - users added
+```
+
+### Operator method
+
+When using operator you need to edit the `kieconfig-<current-version>`, in this case, `7.12.0`.
+With the operator already running in the current namespace, follow the steps below:
+
+
+Create the config-map in the current namespace:
+```bash
+$ oc create configmap postconfigure \
+    --from-file=add-users.cli=add-users.cli \
+    --from-file=delayedpostconfigure.sh=delayedpostconfigure.sh \
+    --from-file=postconfigure.sh=postconfigure.sh
+configmap/postconfigure created
+```
+
+
+Edit the `kieconfigs-7.12.0` config map, you can use either the OCP Web UI or the command line tool, in this example we'll
+use the command line tool:
+
+```bash
+$ oc edit cm kieconfigs-7.12.0
+```
+
+In this case, as we are going to update the KIE Server deployment, you need to update the `servers` section of the
+common.yaml content. If it was for `Business Central, Monitoring or Decision Central`, then the `console` section needs to be updated.
+If it is the `Dashbuilder` then the configMap called `kieconfigs-7.12.0-dashbuilder` needs to be edited.
+
+First, let's locate where is the `servers` section, you can find it around the line **640**.
```

Comment:
I wouldn't put there line number, it can be more confusing once we add some more config and lines will move. And also around line 640 there is config for smart-router and not Servers.
```suggestion
First, let's locate where is the `servers` section.
```

Reply:
+1